samples/c2pa/C2paPlayer/main.js

```diff
@@ -48,7 +48,7 @@ export var C2PAPlayer = function (
 
             //Initialize c2pa timeline and menu
             initializeC2PAControlBar(videoPlayer);
-            initializeC2PAMenu( videoPlayer);
+            initializeC2PAMenu( videoPlayer , c2paMenu);
```

Comment:
I think `c2paMenu` is probably not initialized when we pass it

Reply:
Where does `c2paMenu` get set/initialized?

Reply:
Just a couple of lines below ([line 56](https://github.com/contentauth/dash.js/blob/67bfe6835c67cf0ee88c6e5f6eba2e646240cb6b/samples/c2pa/C2paPlayer/main.js#L56))